src/detail/task_queue.cpp

```diff
@@ -107,4 +107,30 @@ task_queue::~task_queue()
     }
 }
 
+namespace
+{
+auto park_q = boost::lockfree::queue<task_queue *>(32);
```

Comment:
Question: is 32 the maximum queue size or just the initial capacity?

Reply:
Initial capacity